### PR TITLE
Add test for void return type in arrow function

### PR DESCRIPTION
**Pull request by olivernybroe**
Related to: #5235

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -301,7 +301,8 @@
             "Rector\\Utils\\NodeDocumentationGenerator\\Tests\\": "utils/node-documentation-generator/tests",
             "Rector\\Utils\\PHPStanTypeMapperChecker\\": "utils/phpstan-type-mapper-checker/src",
             "Rector\\Utils\\ProjectValidator\\": "utils/project-validator/src",
-            "Rector\\Carbon\\Tests\\": "rules/carbon/tests"
+            "Rector\\Carbon\\Tests\\": "rules/carbon/tests",
+            "Rector\\PostRector\\Tests": "packages/post-rector/tests"
```
Comment (samsonasik) [inline composer.json:305]: need namespace separator:

```suggestion
            "Rector\\PostRector\\Tests\\": "packages/post-rector/tests"
```

Comment (olivernybroe) [reply composer.json]: Yeah 👍

**Comment by samsonasik** [comment]
@olivernybroe I cherry-picked your commit at PR https://github.com/rectorphp/rector/pull/5358

**Comment by olivernybroe** [comment]
@samsonasik Amazing, thanks! I'll close this pr then.

**Pull request closed without merging**